src/uu/tail/src/paths.rs

```diff
@@ -36,6 +26,7 @@ pub struct Input {
 }
 
 impl Input {
+    // TODO: from &str may be the better choice
```

Comment:
Not if the `display_name` field stays a `String`. Although `Into<String>` could work.

Reply:
The field stays a `String`. This is no big change, I just thought accepting `&str` as parameter is the more general and popular approach because a `String` is coercing to `&str` when using a simple & on it. `&String` is also the type we get from `matches` in `Settings::from` when parsing the file arguments to `Input`s, and it seemed more natural to accept a reference as parameter in a `from` method.